modules/oxide/src/less/theme/components/toolbar-button/toolbar-label.less

```diff
@@ -0,0 +1,61 @@
+//
+// Toolbar label
+// When making changes here, please review menubar/menubar.less and menubar/menubar-button.less
+//
+
+@toolbar-label-background-color: @background-color;
+@toolbar-label-border-radius: 3px;
+@toolbar-label-border: 0;
+@toolbar-label-box-shadow: none;
+@toolbar-label-font-size: @font-size-sm;
+@toolbar-label-font-style: normal;
+@toolbar-label-font-weight: @font-weight-normal;
+@toolbar-label-height: 28px;
+@toolbar-label-icon-color: @text-color-muted;
+@toolbar-label-spacing-x: 1px;
+@toolbar-label-spacing-y: 5px;
+@toolbar-label-text-color: @text-color-muted;
+@toolbar-label-text-transform: none;
+@toolbar-label-width: auto;
+
+.tox {
+  legend.tox-label {
```

Comment:
This doesn't march any more?

- [ ] Do we need this CSS what is wrong with the `tox-label` that seems to be used now and is rendering fine, so remove this file?

Reply:
ok, I fixed the CSS instead of removing it, have a check